speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/confscreens/ThresholdConfig.cpp

```diff
@@ -0,0 +1,202 @@
+#include <tgfclient.h>
+#include <iostream>
+#include "ThresholdConfig.h"
+#include "../legacymenu.h"
+#include "ConfigEnums.h"
+#include "Mediator.h"
+
+#define SND_PARAM_FILE    "config/Threshhold.xml"
+#define SND_SCT_THRESHOLD "Threshold Settings"
+#define SND_ATT_ACCEL     "Accel"
+#define SND_ATT_BRAKE     "Brake"
+#define SND_ATT_STEER     "Steer"
+
+static void* s_scrHandle = nullptr;
+static void* s_prevHandle = nullptr;
+
+// Data to store
+float m_accelVal = 0.0f;
+float m_brakeVal = 0.9f;
+float m_steerVal = 0.04f;
```

Comment:
Instead of having a variable for every individual, just use the struct as variable and set those fields. Like
`tThreshold m_decisionThresholds = { DEFAULT_ACCEL, DEFAULT_BRAKE, DEFAULT_STEER };`